apply_extra.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+APP_IMAGE="/app/extra/Trezor-Suite.AppImage"
+
+# Allow image to execute
+
+chmod +x $APP_IMAGE
+
+# Extract image
+
+unappimage $APP_IMAGE
```

Comment:
Looks like this leaks output when installing, can you silence that? It messes with the Flatpak cli.

```
Installing… ████████████████████ 100%  5.2 MB/s  00:00Parallel unsquashfs: Using 12 processors
149 inodes (3156 blocks) to write

[==================================================================================================|] 3156/3156 100%

created 146 files
created 47 directories
        ID                       Branch          Op         Remote               Download
 1. [✓] io.trezor.suite          test            i          suite-origin         170.3 MB / 171.3 MB
```